src/lib.rs

```diff
@@ -1439,3 +1439,10 @@ impl<T: Renderer> Drop for Canvas<T> {
         self.images.clear(&mut self.renderer);
     }
 }
+
+// re-exports
+#[cfg(feature = "image-loading")]
+pub use ::image as img;
```

Comment:
I’ve see other crates re-export the image crate as image. Is it possible to do this here, too?

Reply:
It is not, at least not without some invasive albeit minor refactoring, since we already have a module named image. See my original post in this thread.

Reply:
Ah yes. Sorry, I missed that :)